src/client/Client.js

```diff
@@ -414,6 +415,14 @@ class Client extends BaseClient {
       query.set('guild_id', guildID);
     }
 
+    if (options.additionalScopes) {
+      const scopes = options.additionalScopes;
+      if (!Array.isArray(scopes) || scopes.some(scope => !InviteScopes.includes(scope))) {
+        throw new TypeError('INVALID_TYPE', 'additionalScopes', 'Array of Invite Scopes', true);
+      }
+      query.set('scope', ['bot', ...scopes].join(' '));
```

Comment:
```suggestion
      query.set('scope', `bot ${scopes.join(' ')}`);
```
I think it's better, because it avoids unnecessary iteration, it's not necessarily more readable, it's not necessarily less readable either.
It's still an interesting micro-optimization.